custom_model_runner/datarobot_drum/drum/main.py

```diff
@@ -42,6 +42,22 @@
 import os
 import signal
 import sys
+from datarobot_drum import RuntimeParameters
+
+# Monkey patching for gevent compatibility if running with gunicorn-gevent
+if RuntimeParameters.has("DRUM_SERVER_TYPE") and RuntimeParameters.has(
+    "DRUM_GUNICORN_WORKER_CLASS"
+):
+    if (
+        str(RuntimeParameters.has("DRUM_SERVER_TYPE")).lower() == "gunicorn"
```

Comment:
Maybe define `gunicorn`, `gevent` as constants? THey are reused in several places